code/modules/mob/living/carbon/update_icons.dm

```diff
@@ -155,9 +199,10 @@
 /mob/living/carbon/proc/update_hud_handcuffed()
 	if(hud_used)
 		var/obj/screen/inventory/R = hud_used.inv_slots[slot_r_hand]
-		var/obj/screen/inventory/L = hud_used.inv_slots[slot_l_hand]
-		if(R && L)
+		if(R)
 			R.update_icon()
+		var/obj/screen/inventory/L = hud_used.inv_slots[slot_l_hand]
+		if(L)
 			L.update_icon()

```

Comment:
splitting left and right hand button in case someone makes a freak mob with only a left hand.